setup-env

```diff
@@ -84,7 +84,8 @@ fi
 # Now things are set up to let the OE setup scripts to their jobs
 . $OEROOT/scripts/oe-buildenv-internal &&
     TEMPLATECONF="$TEMPLATECONF" $OEROOT/scripts/oe-setup-builddir || {
-	unset OEROOT DISTRO TEMPLATECONF $TD_VARS
+	unset OEROOT DISTRO TEMPLATECONF
```

Comment:
These probably need to be listed in the `$(echo ...)` in the next line for zsh, or use separate `unset`s for each one.

Reply:
> These probably need to be listed in the `$(echo ...)` in the next line for zsh, or use separate `unset`s for each one.

Yes